projects/common/src/color/color.ts

```diff
@@ -0,0 +1,7 @@
+export enum Color {
```

Comment:
const

Reply:
I made this an enum so that this can also be used as a type for the input params. Please let me know if there's a strong preference to avoid enum here.

Reply:
I think @aaron-steinfeld meant `export const enum`

Reply:
oh. right. ok. now I feel stupid.

Reply:
🦜 (no party parrot to be found)